lib/Serverless.js

```diff
@@ -68,13 +68,13 @@ class Serverless {
   run() {
     if (!this.cli.displayHelp(this.processedInput) && this.processedInput.commands.length) {
       // trigger the plugin lifecycle when there's something which should be processed
-      return this.pluginManager.run(this.processedInput.commands);
+      this.pluginManager.run(this.processedInput.commands);
```

Comment:
Yes. That's also something I was nervous about (never touch a running system :-D)!

The problem is that the return will yield to an exit of the CLI and the tracking happens afterwards.

We could move the tracking before this code and leave this piece of code how it is.

What do you think about that.